pipeline_dp/pld_budget_accounting.py

```diff
@@ -0,0 +1,114 @@
+"""Privacy Loss Distribution (PLD) budget accounting for DP Pipelines."""
+
+from dataclasses import dataclass, field
+from pipeline_dp.aggregate_params import NoiseKind
+from dp_accounting import privacy_loss_distribution
+
+
+@dataclass
+class PLDBudget:
+    _noise_parameter: float = None  # i am not sure how to use this, as noise is calibrated globally?
+
+    @property
+    def noise_parameter(self):
+        if self._noise_parameter is None:
+            raise AssertionError("Noise parameter is not yet defined.")
+        return self._noise_parameter
+
+
+@dataclass
+class RequestedPLDBudget:
+    budget: PLDBudget
+
+
+@dataclass
```

Comment:
I'd say that PLDBudgetAccountant is more like functionality class than data class, so I'd suggest to remove @dataclass.

Reply:
Makes sense, removed 👍